server/api/opportunity/__tests__/opportunity.spec.js

```diff
@@ -23,6 +23,8 @@ import { jwtData } from '../../../middleware/session/__tests__/setSession.fixtur
 test.before('before connect to database', async (t) => {
   t.context.memMongo = new MemoryMongo()
   await t.context.memMongo.start()
+  t.context.people = await Person.create(people).catch((err) => `Unable to create people: ${err}`)
+  t.context.orgs = await Organisation.create(orgs).catch((err) => `Unable to create orgs: ${err}`)
```

Comment:
Similar to my other comment - these `.catch` calls will swallow errors and allow tests to run without the expected data being in the database